BT-Tracking/Services/Store/ScannerStore.swift

```diff
@@ -30,14 +37,19 @@ final class ScannerStore {
     private let didFindSubject = PublishRelay<BTDevice>()
     private let didUpdateSubject = PublishRelay<BTDevice>()
     private let didReceive: Observable<BTDevice>
-    private let timer: Observable<Int> = Observable.timer(.seconds(scanningDelay), period: .seconds(scanningPeriod), scheduler: ConcurrentDispatchQueueScheduler(qos: .background))
+    private let timer: Observable<Int>
     private var period: BehaviorSubject<Void> {
         return BehaviorSubject<Void>(value: ())
     }
     private var currentPeriod: BehaviorSubject<Void>?
     private var devices = [BTDevice]()
     
-    init() {
+    init(scanningPeriod: Int = 60, scanningDelay: Int = 0, dataPurgeInterval: TimeInterval = 14 * 86400) {
+        self.scanningPeriod = scanningPeriod
+        self.scanningDelay = scanningDelay
+        self.dataPurgeInterval = dataPurgeInterval
+        self.timer = Observable.timer(.seconds(scanningDelay), period: .seconds(scanningPeriod), scheduler: ConcurrentDispatchQueueScheduler(qos: .background))
```

Comment:
We need to remove this and keep it 0. This is a delay before the timer fires first event, but is used only once in the begining. I would need to refactor this slightly if we want to support scanning delay between periods. I suggest we don't do it because of issues with passive scanning on iOS.